packages/salesforcedx-utils-vscode/src/output/table.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const COLUMN_SEPARATOR = '  ';
+const COLUMN_FILLER = ' ';
+const HEADER_FILLER = '─';
+
+export interface Row {
+  [column: string]: string;
+}
+
+export interface Column {
+  key: string;
+  label: string;
+}
+
+export class Table {
+  public createTable(rows: Row[], cols: Column[]): string {
+    const maxColWidths = this.calculateMaxColumnWidths(rows, cols);
+
+    let columnHeader = '';
+    let headerSeparator = '';
+    cols.forEach((col, index, arr) => {
+      const width = maxColWidths.get(col.key);
+      if (width) {
+        const isLastCol = index < arr.length - 1;
```

Comment:
This variable naming is a little bit misleading - `isLastCol` is `true` when the index does not correspond to the last column. Maybe isLastCol should equal `index === arr.length-1`, and the logic in line 113 could be based on `!isLastCol`?

Reply:
Ahhh yes that was a hiccup 😛 thanks!